scripts/benchmarks_local.py

```diff
@@ -405,7 +405,7 @@ def generate_single_benchmark_ci_args(parsed_args: Namespace, specific_run_type:
             '--category-exclusion-filter', 'NoInterpreter', 'NoWASM', 'NoMono',
             '--wasmDataDir', os.path.join(get_run_artifact_path(parsed_args, RunType.WasmAOT, commit), "wasm_bundle", "wasm-data"),
             '--wasmEngine', parsed_args.wasm_engine_path,
-            '--wasmArgs', '\"--experimental-wasm-eh --expose_wasm --module\"',
+            '--wasmArgs', '\" --experimental-wasm-eh --expose_wasm --module\"',
```

Comment:
Please put the same comment as in `performance_setup.py` here. Otherwise, I guess, somebody will remove the space.